app/main.ipynb

```diff
@@ -56,23 +80,23 @@
     "![pie chart](../demo/pie.png)"
    ],
    "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
+    "collapsed": false
    }
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
    "source": [
-    "# write your code here"
+    "most_popular_products = df.groupby(\"product\")[\"quantity\"].sum().sort_values(ascending=False)[:10]\n",
+    "total = most_popular_products.sum()\n",
+    "most_popular_products.plot.pie(autopct = lambda p: \"{:.1f}%({:.0f})\".format(p,(p/100)*total), figsize=(10, 8))"
```

Comment:
Commit the notebook with the output cells already executed to show results on github